Firestore/core/test/firebase/firestore/util/async_queue_test.cc

```diff
@@ -87,14 +87,14 @@ TEST_P(AsyncQueueTest, CanScheduleOperationsInTheFuture) {
   std::string steps;
 
   queue.Enqueue([&steps] { steps += '1'; });
+  queue.Enqueue([&steps] { steps += '2'; });
```

Comment:
This is a quick fix to avoid flaky test failures in unrelated PRs.

Reply:
Do you intend to roll this back? It kinda looks like this isn't really testing much anymore. Perhaps you'd be better just commenting it out?

Either way, I'd recommend adding a TODO that indicates what will change after the quick fix is no longer necessary.

Reply:
Added a TODO.